css-two-column-layout/styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  font-family: Karla, sans-serif;
+
+  /* font-style: normal;
+  src: url("https://fonts.gstatic.com/s/karla/v7/qkBbXvYC6trAT7RVLtyU5rZP.woff2") format("woff2"); */
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.header-content {
+  width: 100%;
+  text-align: center;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.content-section-div {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.content-section-one-fourth {
+  flex-basis: calc(100% / 2);
+  display: flex;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.column-one-third {
+  /* flex-basis: calc(100% / 3); */
+}
+
+.column-two-third {
+  flex-basis: calc(200% / 3);
+  margin-top: 20px;
+}
+
+.image-item {
+  display: flex;
+  height: 100px;
+  width: 100px;
+
+  /* height: 50%; */
```

Comment:
If commented code is unnecessary, you should delete it before submitting for review